docs/_sass/components/categories.scss

```diff
@@ -34,6 +34,8 @@
 
   padding: var(--docs-side-padding);
   position: sticky;
+  height: 100vh;
+  overflow: hidden;
```

Comment:
`overflow` should be `auto` instead of `hidden` to make it (categories nav) scrollable